--- bzip2-1.0.5.orig/configure.ac	1970-01-01 01:00:00.000000000 +0100
+++ bzip2-1.0.5/configure.ac	2008-03-22 16:15:55.057937371 +0100
@@ -0,0 +1,9 @@
+AC_INIT([bzip2], [1.0.6])
+AM_INIT_AUTOMAKE
+AC_CONFIG_HEADER([config.h])
+AC_PROG_CC
+AC_PROG_LIBTOOL
+AC_PROG_LN_S
+AC_SYS_LARGEFILE
+AC_CONFIG_FILES([Makefile])
+AC_OUTPUT
--- bzip2-1.0.5.orig/Makefile.am	1970-01-01 01:00:00.000000000 +0100
+++ bzip2-1.0.5/Makefile.am	2008-03-22 16:14:38.913593962 +0100
@@ -0,0 +1,37 @@
+include_HEADERS	= bzlib.h bzlib_private.h
+lib_LTLIBRARIES	= libbz2.la
+bin_PROGRAMS	= bzip2 bzip2recover
+bin_SCRIPTS	= bzdiff bzgrep bzmore
+man_MANS	= bzip2.1 bzgrep.1 bzmore.1 bzdiff.1
+
+libbz2_la_SOURCES	= \
+	blocksort.c \
+	huffman.c \
+	crctable.c \
+	randtable.c \
+	compress.c \
+	decompress.c \
+	bzlib.c \
+	bzlib.h \
+	bzlib_private.h
+libbz2_la_LDFLAGS	= -version-info 1:0:0
+
+bzip2_SOURCES	= bzip2.c
+bzip2_LDADD	= libbz2.la
+
+install-exec-hook:
+	$(LN_S) -f bzip2 $(DESTDIR)$(bindir)/bunzip2
+	$(LN_S) -f bzip2 $(DESTDIR)$(bindir)/bzcat
+	$(LN_S) -f bzgrep $(DESTDIR)$(bindir)/bzegrep
+	$(LN_S) -f bzgrep $(DESTDIR)$(bindir)/bzfgrep
+	$(LN_S) -f bzmore $(DESTDIR)$(bindir)/bzless
+	$(LN_S) -f bzdiff $(DESTDIR)$(bindir)/bzcmp
+
+install-data-hook:
+	echo '.so bzip2.1' >$(DESTDIR)$(mandir)/man1/bzip2recover.1
+	echo '.so bzip2.1' >$(DESTDIR)$(mandir)/man1/bunzip2.1
+	echo '.so bzip2.1' >$(DESTDIR)$(mandir)/man1/bzcat.1
+	echo '.so bzgrep.1' >$(DESTDIR)$(mandir)/man1/bzegrep.1
+	echo '.so bzgrep.1' >$(DESTDIR)$(mandir)/man1/bzfgrep.1
+	echo '.so bzmore.1' >$(DESTDIR)$(mandir)/man1/bzless.1
+	echo '.so bzdiff.1' >$(DESTDIR)$(mandir)/man1/bzcmp.1
--- bzip2-1.0.2/crctable.c	Sun Dec 30 03:19:28 2001
+++ bzip2-1.0.2.new/crctable.c	Fri Feb  1 04:19:11 2002
@@ -58,6 +58,9 @@
   For more information on these sources, see the manual.
 --*/
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
 
 #include "bzlib_private.h"
 
--- bzip2-1.0.2/decompress.c	Sun Dec 30 21:45:53 2001
+++ bzip2-1.0.2.new/decompress.c	Fri Feb  1 04:19:11 2002
@@ -58,6 +58,9 @@
   For more information on these sources, see the manual.
 --*/
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
 
 #include "bzlib_private.h"
 
--- bzip2-1.0.2/dlltest.c	Sun Dec 30 20:44:07 2001
+++ bzip2-1.0.2.new/dlltest.c	Fri Feb  1 04:19:11 2002
@@ -8,6 +8,10 @@
    usage: minibz2 [-d] [-{1,2,..9}] [[srcfilename] destfilename]
 */
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
 #define BZ_IMPORT
 #include <stdio.h>
 #include <stdlib.h>
--- bzip2-1.0.2/randtable.c	Sun Dec 30 03:19:04 2001
+++ bzip2-1.0.2.new/randtable.c	Fri Feb  1 04:19:18 2002
@@ -58,6 +58,9 @@
   For more information on these sources, see the manual.
 --*/
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
 
 #include "bzlib_private.h"
 
--- bzip2-1.0.2/spewG.c	Thu Apr  6 01:24:45 2000
+++ bzip2-1.0.2.new/spewG.c	Fri Feb  1 04:19:18 2002
@@ -9,7 +9,9 @@
    (but is otherwise harmless).
 */
 
-#define _FILE_OFFSET_BITS 64
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
 
 #include <stdio.h>
 #include <stdlib.h>
diff -Nru bzip2-1.0.2/unzcrash.c bzip2-1.0.2.new/unzcrash.c
--- bzip2-1.0.2/unzcrash.c	Thu Apr  6 01:24:46 2000
+++ bzip2-1.0.2.new/unzcrash.c	Fri Feb  1 04:19:18 2002
@@ -13,6 +13,10 @@
    many hours.
 */
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
 #include <stdio.h>
 #include <assert.h>
 #include "bzlib.h"
