diff -urNp -x '*.orig' bzip2-1.0.8.org/Makefile.am bzip2-1.0.8/Makefile.am
--- bzip2-1.0.8.org/Makefile.am	1970-01-01 01:00:00.000000000 +0100
+++ bzip2-1.0.8/Makefile.am	2021-03-21 00:16:33.329899186 +0100
@@ -0,0 +1,37 @@
+include_HEADERS	= bzlib.h bzlib_private.h
+lib_LTLIBRARIES	= libbz2.la
+bin_PROGRAMS	= bzip2 bzip2recover
+bin_SCRIPTS	= bzdiff bzgrep bzmore
+man_MANS	= bzip2.1 bzgrep.1 bzmore.1 bzdiff.1
+
+libbz2_la_SOURCES	= \
+	blocksort.c \
+	huffman.c \
+	crctable.c \
+	randtable.c \
+	compress.c \
+	decompress.c \
+	bzlib.c \
+	bzlib.h \
+	bzlib_private.h
+libbz2_la_LDFLAGS	= -version-info 1:0:0
+
+bzip2_SOURCES	= bzip2.c
+bzip2_LDADD	= libbz2.la
+
+install-exec-hook:
+	$(LN_S) -f bzip2 $(DESTDIR)$(bindir)/bunzip2
+	$(LN_S) -f bzip2 $(DESTDIR)$(bindir)/bzcat
+	$(LN_S) -f bzgrep $(DESTDIR)$(bindir)/bzegrep
+	$(LN_S) -f bzgrep $(DESTDIR)$(bindir)/bzfgrep
+	$(LN_S) -f bzmore $(DESTDIR)$(bindir)/bzless
+	$(LN_S) -f bzdiff $(DESTDIR)$(bindir)/bzcmp
+
+install-data-hook:
+	echo '.so bzip2.1' >$(DESTDIR)$(mandir)/man1/bzip2recover.1
+	echo '.so bzip2.1' >$(DESTDIR)$(mandir)/man1/bunzip2.1
+	echo '.so bzip2.1' >$(DESTDIR)$(mandir)/man1/bzcat.1
+	echo '.so bzgrep.1' >$(DESTDIR)$(mandir)/man1/bzegrep.1
+	echo '.so bzgrep.1' >$(DESTDIR)$(mandir)/man1/bzfgrep.1
+	echo '.so bzmore.1' >$(DESTDIR)$(mandir)/man1/bzless.1
+	echo '.so bzdiff.1' >$(DESTDIR)$(mandir)/man1/bzcmp.1
diff -urNp -x '*.orig' bzip2-1.0.8.org/configure.ac bzip2-1.0.8/configure.ac
--- bzip2-1.0.8.org/configure.ac	1970-01-01 01:00:00.000000000 +0100
+++ bzip2-1.0.8/configure.ac	2021-03-21 00:16:33.329899186 +0100
@@ -0,0 +1,9 @@
+AC_INIT([bzip2], [1.0.6])
+AM_INIT_AUTOMAKE
+AC_CONFIG_HEADER([config.h])
+AC_PROG_CC
+AC_PROG_LIBTOOL
+AC_PROG_LN_S
+AC_SYS_LARGEFILE
+AC_CONFIG_FILES([Makefile])
+AC_OUTPUT
diff -urNp -x '*.orig' bzip2-1.0.8.org/crctable.c bzip2-1.0.8/crctable.c
--- bzip2-1.0.8.org/crctable.c	2019-07-13 19:50:05.000000000 +0200
+++ bzip2-1.0.8/crctable.c	2021-03-21 00:16:33.329899186 +0100
@@ -98,6 +98,9 @@ UInt32 BZ2_crc32Table[256] = {
    0xbcb4666dL, 0xb8757bdaL, 0xb5365d03L, 0xb1f740b4L
 };
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
 
 /*-------------------------------------------------------------*/
 /*--- end                                        crctable.c ---*/
diff -urNp -x '*.orig' bzip2-1.0.8.org/decompress.c bzip2-1.0.8/decompress.c
--- bzip2-1.0.8.org/decompress.c	2019-07-13 19:50:05.000000000 +0200
+++ bzip2-1.0.8/decompress.c	2021-03-21 00:16:33.330899186 +0100
@@ -35,6 +35,9 @@ void makeMaps_d ( DState* s )
       }
 }
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
 
 /*---------------------------------------------------*/
 #define RETURN(rrr)                               \
diff -urNp -x '*.orig' bzip2-1.0.8.org/dlltest.c bzip2-1.0.8/dlltest.c
--- bzip2-1.0.8.org/dlltest.c	2019-07-13 19:50:05.000000000 +0200
+++ bzip2-1.0.8/dlltest.c	2021-03-21 00:16:33.330899186 +0100
@@ -7,6 +7,10 @@
    usage: minibz2 [-d] [-{1,2,..9}] [[srcfilename] destfilename]
 */
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
 #define BZ_IMPORT
 #include <stdio.h>
 #include <stdlib.h>
diff -urNp -x '*.orig' bzip2-1.0.8.org/randtable.c bzip2-1.0.8/randtable.c
--- bzip2-1.0.8.org/randtable.c	2019-07-13 19:50:05.000000000 +0200
+++ bzip2-1.0.8/randtable.c	2021-03-21 00:16:33.330899186 +0100
@@ -78,6 +78,9 @@ Int32 BZ2_rNums[512] = {
    936, 638
 };
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
 
 /*-------------------------------------------------------------*/
 /*--- end                                       randtable.c ---*/
diff -urNp -x '*.orig' bzip2-1.0.8.org/spewG.c bzip2-1.0.8/spewG.c
--- bzip2-1.0.8.org/spewG.c	2019-07-13 19:50:05.000000000 +0200
+++ bzip2-1.0.8/spewG.c	2021-03-21 00:16:33.330899186 +0100
@@ -24,7 +24,9 @@
 	 ------------------------------------------------------------------ */
 
 
-#define _FILE_OFFSET_BITS 64
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
 
 #include <stdio.h>
 #include <stdlib.h>
diff -urNp -x '*.orig' bzip2-1.0.8.org/unzcrash.c bzip2-1.0.8/unzcrash.c
--- bzip2-1.0.8.org/unzcrash.c	2019-07-13 19:50:05.000000000 +0200
+++ bzip2-1.0.8/unzcrash.c	2021-03-21 00:16:33.330899186 +0100
@@ -28,6 +28,10 @@
    ------------------------------------------------------------------ */
 
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
 #include <stdio.h>
 #include <assert.h>
 #include "bzlib.h"
